rosidl_typesupport_c/package.xml

```diff
@@ -22,8 +22,9 @@
   <!-- end of group rosidl_typesupport_c_packages for bloom -->
 
   <buildtool_export_depend>ament_cmake_core</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
```

Comment:
Isn't a runtime dependency on the generator undesired and the whole point of these PRs to avoid that?